CHANGES

```diff
@@ -13,6 +13,12 @@ Deprecated
 Features added
 --------------
 
+* LaTeX: Make the ``toplevel_sectioning`` setting optional in LaTeX theme
+* LaTeX: Allow to override papersize and pointsize from LaTeX themes
+* LaTeX: Add :confval:`latex_theme_options` to override theme options
+* #6740: LaTeX: Support tables without column rules via a theme option:
+  ``show_table_column_rules``
```

Comment:
I'm still thinking about what is the best wording for a vertical line of the table. column rule? vertical rule (vrule)? vertical line (vline)? column separator (colsep)?

Reply:
The reference of booktab package calls it as "rule":

>In British typesetting, a ‘line’ is always called a ‘rule’. Perhaps confusingly (for
historic reasons in fact), the ‘thickness’ of rule is often referred to as is its ‘width’
(whereas just about everyone else would call this ‘depth’ or ‘height’, if they were
thinking of a horizontal rule). A ‘thick black line’ is called a ‘heavy rule’. I have
used this terminology in most of the new commands below. If nothing else it
avoids confusion with \hline.
https://www.ctan.org/pkg/booktabs